### ISSUE TITLE
cellsnp-lite on data from 1 donor

### ISSUE THREAD

**Issue by astaburuaga-garcia** [single-cell-genetics/cellsnp-lite#106]
Hi, I have one bam/barcode file per donor (cell line), and I want to get SNPs from each donor, to then be able to identify the donors in a bam/barcode file where they where multiplexed. Does it make sense to run cellsnp-lite per each donor to identify their SNPs, then merge their vcf files for vireo?

**Comment by hxj5** [comment]
Yes, you may genotype per donor and then combine the donor-wise VCF files for vireo (see [issue 100](https://github.com/single-cell-genetics/cellsnp-lite/issues/100) for detailed discussion).
